query-node/mappings/src/content/video.ts

```diff
@@ -179,6 +181,11 @@ export async function content_VideoCreated(db: DatabaseManager, event: Substrate
   // save video
   await db.save<Video>(video)
 
+  // increase active video counters for channel, channel category, and video category
+  if (isVideoFullyActive(video)) {
+    await updateVideoActiveCounters(db, video, true)
```

Comment:
I've inspected Sumer's runtime, and I think categories and channels can't be changed in this release. I think mappings are aligned with that and correct. A formal load of `video.channel.category` was added, but it will actually have no effect, as you stated. 

In light of that, this PR is actually complete, right?

Reply:
1. You can't set/update channel category, but this is only because those actions are not be processed by the mappings. Channel categories do exist and it's possible to include them in the metadata (`ChannelMetadata.category`), just like video categories. There's nothing in the runtime that prevents this.
2. You can still update video category (`VideoMetadata.category`) in which case you want to reduce the `activeVideosCounter` in the old category (if it exists and the video was active), and possibly increase the `activeVideosCounter` in the new category (if it exists, !== oldCategory and the video is active). This is not correctly handled currently, for example: You have and active video in category `1`, you update its category to `2` and you get invalid state in QN:
  ![image](https://user-images.githubusercontent.com/19978260/147285259-f4e13427-d3ad-40e2-8876-e789dee55fe2.png)

Reply:
Ok. I reworked the feature, so it handles these kinds of updates. There still needs to be some work and especially testing done.